python/src/fast.cpp

```diff
@@ -161,49 +161,6 @@ void init_fast(nb::module_& parent_module) {
             array: The output array.
       )pbdoc");
 
-  m.def(
```

Comment:
I removed the quantize with scales and bias option for now because I don't think it was being used and it was complicating the code slightly. Happy to add it back in later if need be.